src/getters/getConfigFromChain.ts

```diff
@@ -0,0 +1,24 @@
+import { Contract } from 'ethers'
+import { StreamrConfig, streamrConfigABI } from '@streamr/network-contracts'
+import { getSigner } from '~/shared/stores/wallet'
+import { BNish } from '~/utils/bn'
+import { defaultChainConfig } from './getChainConfig'
+
+/**
+ * When we will be needing more of the fields, simply expand the return type and add them
+ */
+export const getConfigFromChain = async (): Promise<{
+    maxPenaltyPeriodSeconds: BNish
+    minimumStakeWei: BNish
+}> => {
+    const signer = await getSigner()
+    const token = new Contract(
+        defaultChainConfig.contracts.StreamrConfig,
+        streamrConfigABI,
+        signer,
+    ) as StreamrConfig
+    return {
+        maxPenaltyPeriodSeconds: await token.maxPenaltyPeriodSeconds(),
```

Comment:
This could use some error handling. Calling the contract methods can throw.

Reply:
I think the only thing we can do is throw an error toast. I'll write something like this in the toast message: "Could not load data from the chain, please try again later"